packages/xrpl/src/utils/isEarlierRippledVersion.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Determines whether a rippled version (source) is earlier than another (target).
+ *
+ * @param source - The source rippled version.
+ * @param target - The target rippled version.
+ * @returns true if source is earlier, false otherwise.
+ */
+// eslint-disable-next-line max-lines-per-function, max-statements -- Disable for this utils functions.
+export default function isEarlierRippledVersion(
```

Comment:
I made it a separate util in case there are features in the future that only apply to a certain build version or later similar to this one. It can certainly be a helper function inside autofill as well

Reply:
Since we are going to remove buildVersion logic anyway then I think we should move this inside `autofill`